framework/src/modules/chain/components/storage/entities/account.js

```diff
@@ -37,6 +37,7 @@ const defaultCreateValues = {
 	nameExist: false,
 	multiMin: 0,
 	multiLifetime: 0,
+	asset: null,
```

Comment:
Maybe makes more sense to initialize with empty object?

Reply:
This fix is applied in 2.1 release